app/(dashboard)/private-cloud/products/page.tsx

```diff
@@ -52,6 +53,7 @@ export default async function ProductsTable({
     cluster,
     userEmail,
     ministryRoles,
+    JSON.parse(active || 'true'),
```

Comment:
Do we need to do JSON.parse here? I believe active will be a string anyways but please verify

Reply:
I wanted to pass a boolean since it makes more sense than passing a string. Another option is to pass the string, then do the parse in the function

Reply:
I see, ok sounds good!

Reply:
Allow me to interject in the review process. It might introduce `errors (JSON parsing error)` if the query parameter `active` is anything other than `true` or `false`. It would be more reliable to directly compare the string values:
1.To set `false` as the default:
- `active === 'true'`

2. To set `true` as the default:
- `active !== 'false'`